style.css

```diff
@@ -1,65 +1,54 @@
 body {
```

Comment:
I think this file is the compiled CSS now that I scrolled down further. If so, it should probably be ignored. You don't want to include compiled files into github, just the raw sources.